tools/bazel.rc

```diff
@@ -2,5 +2,8 @@
 build --verbose_failures
 test --test_output=errors
 
+# Release Build
+build:release -c opt
```

Comment:
I asked in https://github.com/bazelbuild/rules_go/issues/741 but we should use this config seb is added anyway, to change the behavior of kube-inject / to select the optimized sidecar images for instance